src/dotnet/commands/dotnet-new/CSharp_Lib/Library.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace ClassLibrary
+{
+    public class Class1
+    {
+        public void Method1() {
```

Comment:
Seems like the bracing should be consistent in the file.  i.e. `{` should be on a new line.